jep/0000/README.adoc

```diff
@@ -0,0 +1,357 @@
+Java 8 was released in March 2014.
+Java 8 enhancements have stopped as effort focuses on Java 11, Java 17, and beyond.
+
+This Jenkins Enhancement Proposal describes actions required
+to make Jenkins end its support of Java 8 and require Java 11 or newer.
+
+== Specification
+
+The specification has been created as a result of a link:https://groups.google.com/g/jenkinsci-dev/c/YghQ0YP4m78/m/LO9AFa_GAgAJ[Jenkins developers mailing list discussion].
+
+All stories in this JEP have tickets created in Jira or pull requests in GitHub.
+Jira tickets are aggregated in
+link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688].
+
+If the tickets are not linked in the description, they can be found there.
+
+=== Goals and non-goals
+
+Goals:
+
+* Jenkins core development requires Java 11 or newer
+* Jenkins plugin development requires Java 11 or newer when the plugin updates its minimum Java version to a Jenkins core version that does not support Java 8
+
+Non-goals:
+
+* Building all components with Java 11
+* Jenkins replace older API calls with Java 11 API calls
+** Jakarta EE upgrade is not required as part of this JEP
+* All plugins are operational with Java 11
+** Some Jenkins components may be updated to support features offered in Java 11 and beyond, but there is no plan to update all tools
+* Multi-release jar support in development tools
+* Cleanup of removed or deprecated features
+
+=== Scope of changes
+
+* Jenkins core
+* Jenkins remoting
+* Jenkins configuration as code
+* Docker packaging
+* Plugins
+* Jenkins website: www.jenkins.io
+* Jenkins CI instances:
+    link:https://ci.jenkins.io/[ci.jenkins.io],
+    infra.ci.jenkins.io,
+    trusted.ci.jenkins.io,
+    link:https://www.jenkins.io/security/#team[Security team]'s instance
+* Maven build flow (
+    link:https://github.com/jenkinsci/maven-hpi-plugin[Maven HPI Plugin],
+    link:https://github.com/jenkinsci/plugin-pom[Plugin POM],
+    link:https://github.com/jenkinsci/pom[Jenkins POM],
+    link:https://github.com/jenkinsci/bom[Jenkins plugin bill of materials POM],
+    etc.)
+* Test tools:
+    link:https://github.com/jenkinsci/jenkins-test-harness[Jenkins Test Harness (JTH)],
+    link:https://github.com/jenkinsci/acceptance-test-harness[Acceptance Test Harness (ATH)],
+    link:https://github.com/jenkinsci/plugin-compat-tester[Plugin Compatibility Tester (PCT)]
+* User and Developer Documentation
+
+Out of the scope for this JEP:
+
+* Packaging in subprojects (unless specifically noted): Jenkins Operator, Jenkins Evergreen, Jenkinsfile Runner, etc.
+  They will be handled in follow-up JEPs if needed.
+* Gradle build flow
+
+=== Jenkins core patches
+
+Work to be considered is defined in link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688].
+
+==== Library updates
+
+* The link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688 epic] will include library updates as needed.
+* Some updates may require downstream plugin updates.
```

Comment:
noting @jglick's suggestion in https://github.com/jenkinsci/remoting/pull/507#issuecomment-1030629619

may be a better place for it but just recording the feedback:
```suggestion
* Some updates may require downstream plugin updates.
* JavaWS support to be removed from remoting - link:https://github.com/jenkinsci/remoting/pull/507#issuecomment-1030629619[jenkinsci/remoting#507]
```